investment_tracking/models.py

```diff
@@ -27,9 +27,9 @@ class Meta:
         constraints = [
             models.UniqueConstraint(
                 fields=["investor", "startup"],
-                name = "unique_investor_startup"
+                name="unique_investor_startup"
             )
         ]
 
     def __str__(self):
-        return f"Investor: {self.investor.__str__}, StartUp: {self.startup.__str__()}, saved at:{self.saved_at}"
\ No newline at end of file
+        return f"Investor: {self.investor.__str__()}, StartUp: {self.startup.__str__()}, saved at:{self.saved_at}"
```

Comment:
Improvement in __str__ Method
Issue: __str__ methods for InvestorProfile and StartUpProfile are called explicitly.

Improvement: You can call self.investor and self.startup directly, as Django automatically calls __str__() when converting model instances to strings.

def __str__(self):
    return f"Investor: {self.investor}, StartUp: {self.startup}, saved at: {self.saved_at}"

Reply:
Done